### PR TITLE
Run feature tests with Infrataster container

### PR DESCRIPTION
**Pull request by dtan4**
## WHAT
Run feature tests with Infrataster container, which uses [quay.io/wantedly/infrataster](https://quay.io/repository/wantedly/infrataster) image.

I deleted `dotenv` dependency at this PR. We have to specify environment vars by `docker run -e` option from now on (if you need to).

## REF
https://github.com/wantedly/dockerfile-infrataster

### CODE REVIEW

**Comment by dtan4** [comment]
@spesnova Please review this :bow:

File: script/run_tester [hunk 1]
```diff
@@ -7,10 +7,7 @@
 BASE_DIRECTORY=`pwd`
 
 cd ${BASE_DIRECTORY}/test/feature
-docker kill tester > /dev/null 2>&1 || true
```
Comment (spesnova) [inline script/run_tester]: I think you don't need to remove this line :)
Killing and removing container before running is one of docker practice to run successfully :point_up: 
This is referenced to CoreOS's systemd usage.

```
[Unit]
Description=MyApp
After=docker.service
Requires=docker.service

[Service]
TimeoutStartSec=0
ExecStartPre=-/usr/bin/docker kill busybox1
ExecStartPre=-/usr/bin/docker rm busybox1
ExecStartPre=/usr/bin/docker pull busybox
ExecStart=/usr/bin/docker run --name busybox1 busybox /bin/sh -c "while true; do echo Hello World; sleep 1; done"

[Install]
WantedBy=multi-user.target
```
https://coreos.com/docs/launching-containers/launching/getting-started-with-systemd/

Comment (dtan4) [inline script/run_tester]: I see, thanks! I revert this change.

**Comment by spesnova** [comment]
LGTM :+1:
